### DISCUSSION TITLE
Welcome to contrast-ratio-repo Discussions!

### DISCUSSION THREAD

**Discussion by DBenMoshe** [jdwilkin4/contrast-ratio-repo#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dgupta0** [comment]
Hi, 

I am Deeti.  I am Currently learning web development at Free Code Camp and Scrimba. I just completed Javascript and Algorithm course on FCC. I am excited to contribute and learn with everyone on contrast ratio app.  Thanks for having me. Looking forward to building together.
